pkg/mover/chart.go

```diff
@@ -121,14 +137,17 @@ type ChartMoveRequest struct {
 
 // ChartMover represents a Helm Chart moving relocation. It's initialization must be done view NewChartMover
 type ChartMover struct {
+	chartOrigin             string
 	chartDestination        string
 	imageChanges            []*internal.ImageChange
 	chartChanges            []*internal.RewriteAction
 	sourceContainerRegistry internal.ContainerRegistryInterface
 	targetContainerRegistry internal.ContainerRegistryInterface
+	targetOfflineTar        string
```

Comment:
add path suffix

Reply:
not sure what this was about now

Reply:
Add `Path` to the name since it contains the path to the tar

`targetOfflineTarPath` it's just nitpicking

Reply:
Will rename.

Reply:
fixed